backends/tofino/bf-p4c/backend.cpp

```diff
@@ -59,7 +59,7 @@
  *   - BFN::CollectIXBarInfo
  */
 
-#include "backend.h"
```

Comment:
Hmm, I feel like it is more consistent to use the fully qualified path everywhere. And I have seen this done in other projects that do not have their headers separate, too. One benefit I see is that, if you would move your headers into a folder separate from the file, you could have an easier time refactoring.  

A lot of the compiler code is already in this fully-qualified format. Since our formatter moves the main header to the top you can easily identify it. 

Sadly, I do not know a script that fixes the includes automatically, so we currently have a mixture in the code.